internal/commands/image/list_test.go

```diff
@@ -75,19 +75,19 @@ func TestListCmd(t *testing.T) {
 	}{
 		{
 			name: "TestList",
-			expectedOutput: `REPOSITORY                                                       TAG    APP IMAGE ID APP NAME
-foo/bar                                                          <none> 3f825b2d0657 Digested App
-foo/bar                                                          1.0    9aae408ee04f Foo App
-a855ac937f2ed375ba4396bbc49c4093e124da933acd2713fb9bc17d7562a087 <none> a855ac937f2e Quiet App
+			expectedOutput: `REPOSITORY TAG    APP IMAGE ID APP NAME
+foo/bara   <none> 3f825b2d0657 Digested App
```

Comment:
Is `bara` a typo? 🤔

Reply:
It is and it isn't. Technically the test passes because I was messing with another bit of the code to check the build scripts were working. Reverted that bit out now